data/log_level.go

```diff
@@ -0,0 +1,7 @@
+package data
+
+// LogLevels consists of log levels for several types
+var LogLevels = map[string][]string{
+	"syslog": {"EMERG", "ALERT", "CRIT", "ERROR", "WARN", "NOTICE", "INFO", "DEBUG"},
```

Comment:
Nice, but what about having `LogLevel` as a struct with both short (`EMERG`) and long(`EMERGENCY`) values?

Reply:
I'll add soon

Reply:
would could have short and long if both those actually existed and were used. and in the reference of the apache one it doesnt seem to.

Reply:
You right brianvoe. Hmm .. I'm worried about it

Reply:
In that case, how about providing short and long, but both are same as short one.